OpenRA.Mods.Common/Traits/EntersTunnels.cs

```diff
@@ -92,6 +92,11 @@ public override bool CanTargetActor(Actor self, Actor target, TargetModifiers mo
 				if (target == null || target.IsDead)
 					return false;
 
+				var wrapMove = self.TraitOrDefault<IWrapMove>();
+				var move = self.TraitOrDefault<Mobile>();
+				if (move != null && move.IsTraitDisabled && (wrapMove == null || !wrapMove.CanWrapOrder(modifiers)))
```

Comment:
Why do tunnels deserve a special case, but not for example entering a transport?

Reply:
It doesn't make much sense for `AttackMove` to enforce the required modifier in addition to already needing to select "A", but for the other cases this is an oversight. Added the same check in `EnterAlliedActorTargeter`